nodes/src/nodes/eval_cobalt/evaluators/format_check.py

```
@@ -0,0 +1,203 @@
+    """
+    has_code_blocks = '```' in output
+    lines = output.strip().split('\n')
+    non_empty_lines = [line for line in lines if line.strip()]
+
+    if not non_empty_lines:
+        return {'score': 0.0, 'reasoning': 'Output is empty.'}
+
+    if has_code_blocks:
+        return {'score': 1.0, 'reasoning': 'Output contains code blocks.'}
+
+    indented_lines = sum(1 for line in non_empty_lines if line.startswith('    ') or line.startswith('\t'))
```

Ruff (0.15.10)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)